core/network/impl/protocols/block_announce_protocol.cpp

```diff
@@ -120,12 +123,33 @@ namespace kagome::network {
     if (not seen_.add(peer_id, block_announce.header.hash())) {
       return true;
     }
-    main_pool_handler_->execute(
```

Comment:
Revert `BlockAnnounceProtocol`.

This feature should be implemented in `SynchronizerImpl`.
It selects which blocks to fetch from which peers (see `askNextPortionOfBlocks`).